libraries/AP_Mount/AP_Mount_Tusuav.cpp

```diff
@@ -0,0 +1,1128 @@
+                    _rangefinder_dist_m = temp * 0.1f;
+                    break;
+                }
+
+                case GimbalSubCmd::GimbalSubCmd_GimbalSn:{
+					// gimbal model, length is over 14 bytes and less than 18 bytes
+                    if(_parsed_msg.msg.data_len >= 14){
+                        _got_model_name = true;
+                        if(_parsed_msg.msg.data_len > 18){
+                            _parsed_msg.msg.data_len = 18;
+                        }
+                        
+                        strncpy((char *)_model_sn, (const char *)_parsed_msg.msg.data, _parsed_msg.msg.data_len);
+                        GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s SN: %s", send_text_prefix, (const char*)_model_sn);
+                    }
+                    break;
+                }
+
+                default:break;
+                
+            }
+            break;
+        }
+
+        case GimbalCmd::MasCmd_Device: {
+            switch ((DeviceSubCmd)_parsed_msg.msg.sub_cmd) {
+                case DeviceSubCmd::Firmware_Info: {
+                    if (_parsed_msg.msg.data_len >= 10) { 
+                          _got_firmware_version = true;
+
+						// print both app&boot version and compiling time in UTC+8
+                        GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s AppVersion: %d.%d.%d %02d-%02d-%02d %02d:%02d", send_text_prefix, 
+                                (unsigned char)_parsed_msg.msg.data[0],  
+                                (unsigned char)_parsed_msg.msg.data[1],  
+                                (unsigned char)_parsed_msg.msg.data[2],  
+                                (unsigned char)_parsed_msg.msg.data[3],  
+                                (unsigned char)_parsed_msg.msg.data[4],  
+                                (unsigned char)_parsed_msg.msg.data[5], 
+                                (unsigned char)_parsed_msg.msg.data[6],
+                                (unsigned char)_parsed_msg.msg.data[7]); 
+                                    
+                        GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s BootVersion: %d.%d.%d %02d-%02d-%02d %02d:%02d", send_text_prefix,                                    
+                                (unsigned char)_parsed_msg.msg.data[8],
+                                (unsigned char)_parsed_msg.msg.data[9],
+                                (unsigned char)_parsed_msg.msg.data[10],
+                                (unsigned char)_parsed_msg.msg.data[11],
+                                (unsigned char)_parsed_msg.msg.data[12],
+                                (unsigned char)_parsed_msg.msg.data[13],
+                                (unsigned char)_parsed_msg.msg.data[14], 
+                                (unsigned char)_parsed_msg.msg.data[15]);
+                         break;
+                    }
+                    break;
+                }
+                default:break;
```

Comment:
we normally separate these into two lines

Reply:
Thank you for your reply. I will make modifications based on your suggestions. Another question, can I continue submitting the modified code? Do I need to create a new pull request?

Reply:
Hi @sunny-1987,

feel free to keep updating this PR.  No need to create a new one.  By the way, if you think we're going to slow in reviewing, the best thing to do is post a comment asking that the PR be added to the weekly dev call.  You can even attend if you like it is 8am Japan time each tuesday.  If that's too early then there's an "EU Dev Call" on Wed at 4pm (Japan Time).

Reply:
Hi! Could you let me know how to join the "EU Dev Call" on Wednesdays? Thanks!

Reply:
Hi @sunny-1987, just join ArduPilot's discord server and then go to the "weekly-devcall" voice channel at 4pm Japan time today.

Reply:
ok,thanks.

Reply:
What is your discord handle @sunny-1987 ? I'd like to try to contact you while I'm in China.